frame/offences/benchmarking/src/lib.rs

```diff
@@ -73,21 +93,27 @@ fn create_offender<T: Trait>(n: u32, nominators: u32) -> Result<T::AccountId, &'
 	// Create n nominators
 	for i in 0 .. nominators {
 		let nominator_stash: T::AccountId = account("nominator stash", n * MAX_NOMINATORS + i, SEED);
+		let nominator_stash_lookup: <T::Lookup as StaticLookup>::Source =
+			T::Lookup::unlookup(nominator_stash.clone());
 		let nominator_controller: T::AccountId = account("nominator controller", n * MAX_NOMINATORS + i, SEED);
-		let nominator_controller_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(nominator_controller.clone());
+		let nominator_controller_lookup: <T::Lookup as StaticLookup>::Source =
```

Comment:
optionally: I would spice things up here with a `type BalanceOf<T>` and `LookupOf<T>`.